core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c

```diff
@@ -3,13 +3,86 @@
  * Copyright (c) 2018-2019, STMicroelectronics
  */
 
+#include <drivers/stm32mp1_pwr.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <kernel/panic.h>
 #include <mm/core_memprot.h>
 #include <platform_config.h>
-#include <drivers/stm32mp1_pwr.h>
+
+#define PWR_CR3_USB33_EN	BIT(24)
+#define PWR_CR3_USB33_RDY	BIT(26)
+#define PWR_CR3_REG18_EN	BIT(28)
+#define PWR_CR3_REG18_RDY	BIT(29)
+#define PWR_CR3_REG11_EN	BIT(30)
+#define PWR_CR3_REG11_RDY	BIT(31)
+
+struct pwr_regu_desc {
+	unsigned long level_mv;
```

Comment:
I'm curious about the choice of type. Why `unsigned long` and not `unsigned int` or `uint32_t`?

Reply:
No strong constraint here. `unsigned int` would fit.
I'll fix as it looks more... simple.